packages/ui/components/NotificationBanner/NotificationBanner.tsx

```diff
@@ -9,12 +9,13 @@ type NotificationBannerProps<C extends ElementType = ElementType> = PolymorphicC
     success?: boolean
     children: ReactNode
     className?: string
+    richText?: boolean
   }
 >
 
 export const NotificationBanner = forwardRef(
   <C extends React.ElementType = 'div'>(
-    { as, heading, children, success, className, ...props }: NotificationBannerProps<C>,
+    { as, heading, children, success, className, richText, ...props }: NotificationBannerProps<C>,
```

Comment:
default to false so no other `NotificationBanner`s are effected

`isRichText = false`